packages/client/components/DashTopBar.tsx

```diff
@@ -35,9 +35,15 @@ const Wrapper = styled('header')({
 })
 
 const LeftNavToggle = styled(PlainButton)({
+  borderRadius: 100,
   fontSize: ICON_SIZE.MD24,
   lineHeight: '16px',
-  paddingLeft: 16
+  paddingLeft: 16,
```

Comment:
Here `paddingLeft` gets overridden by `padding` that comes after it, so it should be removed.

MD icons in the top bar have a `24px` footprint. Here you are adding some padding so that you can have a box-shadow as a circle. This is fine as long as the 24px footprint of the icon is centered vertically and spaced correctly horizontally. For the left nav icon, it needs to be centered in a 56x56 area. The left and right gutters need to be 16 to line up with the navigation items below. I believe the margin can be set to 12 instead of 8 to line it up. (margin 12 plus padding 4 gives us the 16 on both sides)